build_tools/cmake.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+SCRIPT_PATH=$( cd "$(dirname "${BASH_SOURCE}")" ; pwd -P )
+if [ -d "$SCRIPT_PATH/../_build/" ]; then rm -r "$SCRIPT_PATH/../_build/"; fi
+mkdir "$SCRIPT_PATH/../_build/"
+[ -e "$SCRIPT_PATH/../build.log" ] && rm "$SCRIPT_PATH/../build.log"
+echo "Cmaking Seashell to \`/_build\`... logging to /build.log"
+cd "$SCRIPT_PATH/../_build" && cmake "$SCRIPT_PATH/../" -DCMAKE_INSTALL_PREFIX="$SCRIPT_PATH/../_install"  > ../build.log 2>&1
```

Comment:
2 comments:
 - You probably want to set LLVM_TARGETS_TO_BUILD=arch/native (can't remember the appropriate one) to speed up the build.
 - Maybe also accept additional arguments to CMake via environment variables/command line.